dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/impl/MediaTypeHeaderProvider.java

```diff
@@ -90,9 +90,8 @@ public static MediaType internalValueOf(String mType) {
         int paramsStart = mType.indexOf(';', i + 1);
         int end = paramsStart == -1 ? mType.length() : paramsStart;
 
-        String type = mType.substring(0, i);
-        String subtype = mType.substring(i + 1, end);
-        //Liberty change start
+        String type = mType.substring(0, i).trim();
+        String subtype = mType.substring(i + 1, end).trim();
```

Comment:
Looks like you removed the `//Liberty change start`, but not the `//Liberty change end`

Reply:
Good catch... done.